app/services/credit_notes/create_from_termination.rb

```diff
@@ -79,7 +81,9 @@ def remaining_duration
         end
       end
 
-      (to_date - billed_from).to_i
+      duration = (to_date - billed_from).to_i
+
+      duration < 0 ? 0 : duration
```

Comment:
Style/TernaryParentheses: Use parentheses for ternary expressions with complex conditions.